tests/roles/dataplane_adoption/tasks/octavia_certs.yaml

```diff
@@ -0,0 +1,80 @@
+- name: Migrate old Octavia certificates to new dual CA structure
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    CONTROLLER1_SSH="{{ controller1_ssh }}"
+    CONTROLLER1_SCP="{{ controller1_ssh  | regex_replace('^ssh', 'scp')}}"
+    SERVER_CA_PASSPHRASE=$($CONTROLLER1_SSH grep ^ca_private_key_passphrase /var/lib/config-data/octavia/etc/octavia/octavia.conf)
```

Comment:
I'm wondering if the file from **config-data/puppet-generated** would be more appropriate here

Reply:
I changed the path now. It is the same file, and I noticed that some other certificate related files were also stored in different locations and I wasn't always sure which location is best. Again, I'll push the change soon.